CODEOWNERS

```diff
@@ -0,0 +1,30 @@
+# CODEOWNERS for autoreview assigning in github
+
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# Root folder
+/CODEOWNERS                               @carlescufi
+/LICENSE                                  @carlescufi
+/Jenkinsfile                              @carlescufi
+/README.rst                               @ru-fu @carlescufi
+
+# All cmake related files
+/CMakeLists.txt                           @SebastianBoe
+
+# All Kconfig related files
+Kconfig*                                  @SebastianBoe
+
+# All documentation related files
+*.rst                                     @ru-fu
+
+# All subfolders
+/ble_controller/                          @joerchan @rugeGerritsen
+/ble_controller/**/*.rst                  @b-gent
+/bsdlib/                                  @rlubos @gbakke
```

Comment:
@b-gent worked with bsdlib documentation, I'd add him to cover *.rst files in bsdlib as well.